renovate.json

```diff
@@ -66,12 +66,14 @@
             "separateMajorMinor": false
         },
         {
+            "assignees": ["jxn-30"],
```

Comment:
Maybe @Suchty112  as well?

Reply:
I would like to keep an eye on engine updates myself to be up-to-date about the versions ;)